packages/plugins/rubygem-smart-proxy-probing/rubygem-smart-proxy-probing.spec

```diff
@@ -1,44 +1,51 @@
-# Generated from smart-proxy-probing-0.0.2.gem by gem2rpm -*- rpm-spec -*-
 # template: smart_proxy_plugin
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%{!?_root_datadir:%global _root_datadir %{_datadir}}
+%{!?_root_localstatedir:%global _root_localstatedir %{_localstatedir}}
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+
 %global gem_name smart-proxy-probing
 %global plugin_name smart-proxy-probing
 
-%global foreman_proxy_min_version 1.17
-%global foreman_proxy_dir %{_datarootdir}/foreman-proxy
+%global foreman_proxy_min_version 1.24
```

Comment:
this should be 1.25 now, right? as 1.24 does not have an SCL'ed proxy.

Reply:
Since it's just updated the specs to an scl format, but not building in the scl tags, would that make a difference?  The new spec should still be compatible with 1.24?

Reply:
you're right, but then it's also compatible with 1.17 ;-)